sim_node:
  ros__parameters:
    step_rate: 1000.0
    pub_rate: 50.0
    twist_reference_max_oldness: 1.0
    be_reference_clock: false
    simulate_localization: true
    model: bicycle

    model_params:
      bicycle:
        base_link_offset: 0.0
        wheel_base: 2.0
        drive_on_steered_wheel: yes
        reverse: false
        drive_actuator:
          dead_time: 0.2
          time_constant: 0.2
          max_acceleration: 1.0
          max_velocity: 2.0
        steering_actuator:
          dead_time: 0.2
          time_constant: 0.05
          max_velocity: 2.0
          max_position: 0.0
      differential:
        base_link_offset: 0.0
        track: 0.8
        drive_actuators:
          dead_time: 0.2
          time_constant: 0.2
          max_acceleration: 1.0
          max_velocity: 0.5

    localization:
      odom_walk_velocity_translation: 0.0025
      odom_walk_velocity_rotation: 0.0076
      map_sample_noise_translation: 0.000025
      map_sample_noise_rotation: 3.1e-6
